features/backend/zones.feature

```diff
@@ -25,47 +25,38 @@ Feature: Zones
           And I am on the zone index page
          Then I should see "There are no zones configured"
 
+    @javascript
     Scenario: Accessing the zone creation form
-        Given I am on the dashboard page
-         When I follow "Zones"
-          And I follow "Create zone"
-         Then I should be on the zone creation page
+        Given I am on the zone index page
+        And I select "Country" from "form_type"
```

Comment:
Yes, I am fine with adding a custom step for that:

``And I want to create a :type zone type`` or something like that.